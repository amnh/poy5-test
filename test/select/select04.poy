(* 'select' terminals based terminal names 1 *)
set(seed:1)
read("1.fas")
select(names:("t3","t5","t7","t9","t11","t13"))
build(2)
report(data)