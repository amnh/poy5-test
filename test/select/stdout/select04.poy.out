Data
        
        
         Taxa
             
             Total 6
                                                        
                     Name         Code          
                    t11          8            {}
                    t13          6            {}
                    t3           3            {}
                    t5           5            {}
                    t7           7            {}
                    t9           4            {}
         Synonyms
                     
                     
                      
                 
    Ignored taxa
        t8; t6; t49; t48; t47; t46; t45; t44; t43; t42; t41; t40; t4; 
        t39; t38; t37; t36; t35; t34; t33; t32; t31; t30; t29; t28; t27
        ; t26; t25; t24; t23; t22; t21; t20; t2; t19; t18; t17; t16; 
        t15; t14; t12; t10; t1; 
    Characters
        Non Additive
            
            Total 0
                   
            
        Additive
            
            Total 0
                   
            
        Sankoff
            
            Total 0
                   
            
        Molecular
            
            Total 5
                                                                                                                                                         
                    Name         Initial Assignment         Transformation Cost Matrix         Gap Opening         Weight         class          
                   1.fas:0      Direct Optimization        tcm:(1,2)                          0                   1.             sequence      {}
                   1.fas:1      Direct Optimization        tcm:(1,2)                          0                   1.             sequence      {}
                   1.fas:2      Direct Optimization        tcm:(1,2)                          0                   1.             sequence      {}
                   1.fas:3      Direct Optimization        tcm:(1,2)                          0                   1.             sequence      {}
                   1.fas:4      Direct Optimization        tcm:(1,2)                          0                   1.             sequence      {}
            
        
    Ignored characters
        
    Files
             
             
              File
                      
                      
                      File Name : 1.fas
                       File Contents 
                                     
                                      Characters
                                    
                  
         




